EasyClangComplete.py

```diff
@@ -210,6 +214,37 @@ def config_removed(future):
         elif future.cancelled():
             log.debug(" could not remove config -> cancelled")
 
+    @staticmethod
+    def on_open_declaration(location):
+        """Callback called when link to type is clicked in info popup
+
+        Opens location with type declaration
+
+        """
+        sublime.active_window().open_file(location, sublime.ENCODED_POSITION)
+
+    def info_finished(self, future):
+        """Callback called when additional information for tag is available.
+
+        Creates popup containing information about text under the cursor
+
+        """
+        if not future.done():
+            return
+        (completion_request, result) = future.result()
```

Comment:
`completion_request` is a wrong name. This function has nothing to do with completion.